spec/support/content_data_api.rb

```diff
@@ -211,7 +211,7 @@ def default_previous_single_page_payload(base_path, from, to)
             },
             {
               name: "pviews",
-              total: 30,
+              total: 30000,
```

Comment:
Thanks @cbaines. The motivation was to both create more realistic test data and to make sure we have checks around the way we format data. We had a bug, due to the way we use `number_with_delimiter`, that wasn't represented in our tests because the values in our test data was all <1000 which meant that no commas were included in the numbers. (The bug was fixed [here](https://github.com/alphagov/content-data-admin/pull/151)).

Sorry for the lack of explanation in this PR.